tests/wpt/web-platform-tests/html/infrastructure/common-dom-interfaces/collections/radionodelist.html

```diff
@@ -66,8 +66,12 @@
   assert_true(document.getElementById("r2").checked,
               "The second radio input element should be checked.");
 
-  document.getElementById("r1").removeAttribute("value");
-  rdoList.value = "on";
+	document.getElementById("r1").removeAttribute("value");
+	// assert_equals(document.getElementById("r1").getAttribute("value"), -39);
+	// assert_equals(document.getElementById("r1").value, -39); // but got string "1"
+
+	rdoList.value = "on";
+	// expected "1" but got "2"
```

Comment:
I will.  This is a WIP branch.

I just thought it was interesting, because it seems like IDL attribute has the right value before it is set to "on".